.editorconfig

```diff
@@ -61,6 +61,9 @@ trim_trailing_whitespace = unset
 [nixos/modules/services/networking/ircd-hybrid/*.{conf,in}]
 trim_trailing_whitespace = unset
 
+[nixos/modules/services/video/epgstation/streaming.json]
+insert_final_newline = unset
```

Comment:
Why can't the file have a final newline?

Reply:
It's an auto-generated file and should never be edited by hand, so I thought it'd be easier to add exceptions rather than perform ad-hoc adjustments to the generated output each time it trip ups EditorConfig.